src/platform/packages/shared/kbn-discover-contextual-components/src/data_types/logs/components/summary_column/summary_column.test.tsx

```diff
@@ -46,7 +47,15 @@ const getSummaryProps = (
   isDetails: false,
   row: record,
   dataView: dataViewMock,
-  fieldFormats: fieldFormatsMock,
+  fieldFormats: {
+    ...fieldFormatsMock,
+    getDefaultInstance: jest
+      .fn()
+      .mockImplementation((...params: Parameters<IFieldFormatsRegistry['getDefaultInstance']>) => ({
+        ...fieldFormatsMock.getDefaultInstance(...params),
+        convert: jest.fn().mockImplementation((t: string) => String(t)),
```

Comment:
Is there a difference between `String(t)` and \`${t}\`? There are some particularities in using `String` directly, as `new String(t)` is not the same as `String(t)`, and maybe to avoid potential ambiguities, we could just use template string literals to sidestep that completely?

Reply:
I think with "String" it's an explicit conversion, but apart from that ,there seems to be no real difference in the result between both ways of conversion, only thing I found is, that `${t}` doesn't seem to work with Symbols:

![CleanShot 2025-02-17 at 17 21 34@2x](https://github.com/user-attachments/assets/3f7df3dd-3e8a-434c-a810-c38cfa5ff23e)

Reply:
Interesting, but then `String` with `Symbol` works in coercing it into a string, but not template string literals? JS is a mess. Oh well, TIL.

Reply:
yes, good old `String` deals with it without exception